forem/forem
# Remove admin_member_view Feature Flag-Related Conditionals

## What type of PR is this? (check all applicable)


## Description
This PR removes conditional code related to the `admin_member_view` feature flag now that the feature has been officially launched. 

To be more specific, this PR does the following:

- [x] Adds a DUS that both disables and removes the `admin_member_view` feature flag
- [x] Updates `Admin::UsersController#credit_params` 
- [x] Updates views that relied on the `edit_admin_user_path`
- [x] Updates specs that relied on the old Admin user view or the `edit_admin_user_path`
- [x] Removes some unused views and specs (follow up PR that addresses the rest upcoming!)

## Related Tickets & Documents
Closes #16849 

## QA Instructions, Screenshots, Recordings
Ensure that all checks pass. ♻️  Additionally, click around the Admin Member View and ensure that it still works as expected.

### UI accessibility concerns?
N/A

## Added/updated tests?

- [x] Yes - I've updated, removed, and added tests.

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![Mary Poppins Flying Away via Her Umbrella with the Text "So Done."](https://media.giphy.com/media/wzD3nQPA4gqHK/giphy.gif)
